exe/miq

```diff
@@ -113,6 +113,8 @@ module Miq
   class Generate < Thor
     desc "lwimiq", "Generate a LWIMIQ blog post"
     option :current
+    option :author, :default => '<author>'
```

Comment:
:thinking: we could try to derive the author's name with something like:

```bash
git remote get-url origin | sed s/git@github.com:// | cut -d\/ -f1
```

Reply:
@imtayadeway - Were you thinking backticks for this?  - I like it a lot - but wasn't sure on the best way to add it in.